ocp4/profiles/coreos-ncp.profile

```diff
@@ -140,75 +133,75 @@ selections:
 
     ### Kernel Config
     ## Boot prompt
-    #- grub2_audit_argument
-    #- grub2_audit_backlog_limit_argument
-    #- grub2_slub_debug_argument
-    #- grub2_page_poison_argument
-    #- grub2_vsyscall_argument
-    #- grub2_vsyscall_argument.role=unscored
-    #- grub2_vsyscall_argument.severity=info
-    #- grub2_pti_argument
+    - grub2_audit_argument
+    - grub2_audit_backlog_limit_argument
+    - grub2_slub_debug_argument
+    - grub2_page_poison_argument
+    - grub2_vsyscall_argument
+    - grub2_vsyscall_argument.role=unscored
+    - grub2_vsyscall_argument.severity=info
+    - grub2_pti_argument
 
     ## Security Settings
-    #- sysctl_kernel_kptr_restrict
-    #- sysctl_kernel_dmesg_restrict
-    #- sysctl_kernel_kexec_load_disabled
-    #- sysctl_kernel_yama_ptrace_scope
-    #- sysctl_kernel_perf_event_paranoid
-    #- sysctl_user_max_user_namespaces
-    #- sysctl_user_max_user_namespaces.role=unscored
-    #- sysctl_user_max_user_namespaces.severity=info
-    #- sysctl_kernel_unprivileged_bpf_disabled
-    #- sysctl_net_core_bpf_jit_harden
+    - sysctl_kernel_kptr_restrict
+    - sysctl_kernel_dmesg_restrict
+    - sysctl_kernel_kexec_load_disabled
+    - sysctl_kernel_yama_ptrace_scope
+    - sysctl_kernel_perf_event_paranoid
+    - sysctl_user_max_user_namespaces
+    - sysctl_user_max_user_namespaces.role=unscored
+    - sysctl_user_max_user_namespaces.severity=info
+    - sysctl_kernel_unprivileged_bpf_disabled
+    - sysctl_net_core_bpf_jit_harden
 
     ## File System Settings
-    #- sysctl_fs_protected_hardlinks
-    #- sysctl_fs_protected_symlinks
+    - sysctl_fs_protected_hardlinks
+    - sysctl_fs_protected_symlinks
 
     ### Audit
-    #- var_auditd_flush=incremental_async
-    #- auditd_data_retention_flush
-    #- auditd_local_events
-    #- auditd_write_logs
-    #- auditd_log_format
-    #- auditd_freq
-    #- auditd_name_format
-    #- var_auditd_action_mail_acct=root
-    #- var_auditd_space_left_action=email
-    #- auditd_audispd_configure_remote_server
-    #- auditd_audispd_encrypt_sent_records
-    #- auditd_audispd_disk_full_action
-    #- auditd_audispd_network_failure_action
-
+    - service_auditd_enabled
+    - var_auditd_flush=incremental_async
+    - auditd_data_retention_flush
+    - auditd_local_events
+    - auditd_write_logs
+    - auditd_log_format
+    - auditd_freq
+    - auditd_name_format
+    - var_auditd_action_mail_acct=root
+    - var_auditd_space_left_action=email
+    - auditd_audispd_configure_remote_server
```

Comment:
Note that with OCP, we'll be using the cluster-logging-operator to forward the logs off the cluster.

Reply:
Showing my lack of product knowledge here - does that operator also pickup the CoreOS logs?

Reply:
The operator doesn't do it. We rely on the Cluster Logging Operator. audispd is not installed in CoreOS.

Reply:
@jhrozek @JAORMX is the cluster operator using FIPS libraries and encryptions, and is it realtime off loading?

Reply:
It's real time off loading, I'll get back to you on the FIPS part

Reply:
Fluentd uses OpenSSL underneath, so it should be FIPS compliant.

Reply:
@JAORMX one more question I forgot to ask. How is fluentd getting the audit logs from a node?

Reply:
Fluentd runs in a pod on the node and reads the file. It subsequently forwards whatever is written to that file. It asistió keeps track of the last position it read.